packages/blueprints-integration/src/api/system.ts

```diff
@@ -1,12 +1,45 @@
+import type { IBlueprintTriggeredActions } from '../triggers'
 import type { MigrationStepSystem } from '../migrations'
 import type { BlueprintManifestBase, BlueprintManifestType } from './base'
+import type { ICoreSystemApplyConfigContext } from '../context/systemApplyConfigContext'
+import type { ICoreSystemSettings } from '@sofie-automation/shared-lib/dist/core/model/CoreSystemSettings'
 
 export interface SystemBlueprintManifest extends BlueprintManifestBase {
 	blueprintType: BlueprintManifestType.SYSTEM
 
-	/** A list of Migration steps related to the Core system */
+	/** A list of Migration steps related to the Core system
+	 * @deprecated This has been replaced with `validateConfig` and `applyConfig`
```

Comment:
Yeah, `validateConfig` doesn't make sense currently, because there is no blueprint-config blob related to the system blueprint. Now that I think about it, even the name `applyConfig` is not exactly accurate to what it does, but it is consistent with the other blueprints.

Which has made a few bits of mess around the place trying to avoid reinventing an almost identical flow. I am trying to avoid making it too distinct because I suspect that over time some uses cases for that config will reveal itself, so being able to add in the skipped pieces would be really nice.

I'll update the comment, and remove the commented out methods below